internal/bzlmod/utils.bzl

```diff
@@ -68,7 +68,8 @@ def get_directive_value(directives, key):
     value = None
     for directive in directives:
         if directive.startswith(prefix):
-            value = directive[len(prefix):]
+            # Treat "gazelle:key    value" the same as "gazelle:key value".
+            value = directive[len(prefix):].lstrip()
```

Comment:
As mentioned earlier, Gazelle also accepts trailing whitespace. It looks like you decided against doing so here, but I just want to confirm that you did so consciously.

Reply:
Yes, I did s consciously. The reason is that trailing whitespace in string list elements is very easy to spot whereas duplicated whitespace in their middle is not. We could always relax this later if there is a need.